src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -90,7 +90,7 @@ public function load(array $configs, ContainerBuilder $container): void
         $javascript = $this->buildJavascripts($config);
 
         $config['assets']['stylesheets'][] = \sprintf(
-            'bundles/sonataadmin/admin-lte-skins/%s.min.css',
+            'bundles/sonataadmin/admin-lte-skins/%s.min.css?v=4.36',
```

Comment:
Not sure we need versioning for a dependency of sonata admin, the main issue is about our own css